Library/Homebrew/cask/cask.rb

```diff
@@ -165,6 +166,10 @@ def installed?
       !versions.empty?
     end
 
+    def loaded_from_api?
```

Comment:
Could also probably use `attr_predicate` here

Reply:
Interesting, I didn't realize that was a thing.